### DISCUSSION TITLE
NFT Challenges

### DISCUSSION THREAD

**Discussion by Anshpatwal** [Cyfrin/foundry-full-course-cu#2430] [General]
Hey guys, i just completed Remix Storage Factory and i want to try out challenges that give NFT Rewards , But The link redirects me to Etherscan and i kinda dont know how to take up the challenge , so if you know the please help out.

**Comment by EngrPips** [comment]
Hello @Anshpatwal, read this discussion https://github.com/Cyfrin/foundry-full-course-cu/discussions/2018

**Reply by Anshpatwal** [reply]
Thanks Bro you always help! @EngrPips

**Reply by Anshpatwal** [reply]
I read the discussion(still dumbfolded) and i think i need to pay some gas on ZKsync mainnet for the transaction.

**Reply by EngrPips** [reply]
You want to mint on zksync-era?

**Reply by Anshpatwal** [reply]
Yes, on seopila too

**Reply by EngrPips** [reply]
Oh I see, you will pay some gas for the transaction

**Reply by Anshpatwal** [reply]
> Oh I see, you will pay some gas for the transaction

Well i am kinda Broke 💀 so , Imma skip that

**Reply by EngrPips** [reply]
Do you know that you can get the eth to pay gas fee on sepolia for free here https://faucets.chain.link
